<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/img.qrc">
    <normaloff>:/files/profits_32.png</normaloff>:/files/profits_32.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(254, 254, 254);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>159</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;
border-radius:10px;
color:black;
border:1px solid #dcdcdc;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>159</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;
border-radius:10px;
color:black ;
border:1px solid #dcdcdc;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>159</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;
border-radius:10px;
color:black;
border:1px solid #dcdcdc;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>159</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;
border-radius:10px;
color:black;
border:1px solid #dcdcdc;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="date_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>179</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(22, 22, 22);
background:transparent</string>
    </property>
    <property name="text">
     <string>date</string>
    </property>
   </widget>
   <widget class="QLabel" name="date_aquisition">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>199</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>179</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:black</string>
    </property>
    <property name="text">
     <string>materiel</string>
    </property>
   </widget>
   <widget class="QLabel" name="materiel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>199</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:black</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>179</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:black</string>
    </property>
    <property name="text">
     <string>dur√©e d' utilisation</string>
    </property>
   </widget>
   <widget class="QLabel" name="duree">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>199</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:black</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>179</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:black</string>
    </property>
    <property name="text">
     <string>taux d'amortissement</string>
    </property>
   </widget>
   <widget class="QLabel" name="taux_amortissement">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>199</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:black</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>211</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:black;</string>
    </property>
    <property name="text">
     <string>tableau d'amortissement</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_amortissement">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>1041</width>
      <height>16</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::SelectedClicked</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>true</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>10</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>35</number>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="prix">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>79</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background:white;
border:1px solid #dcdcdc;
border-radius:5px
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="materiel_selection">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>79</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;
color:black;
border:1px solid #dcdcdc;
border-radius:5px;
</string>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QDateEdit" name="date_selection">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>79</y>
      <width>221</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background:white;
border:1px solid #dcdcdc;
border-radius:5px</string>
    </property>
   </widget>
   <widget class="QLabel" name="location">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>24</y>
      <width>211</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
</string>
    </property>
    <property name="text">
     <string>amortissement calculator</string>
    </property>
   </widget>
   <widget class="QComboBox" name="amort_selection">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>79</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;
color:black;
border:1px solid #dcdcdc;
border-radius:5px;
</string>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>lin√©aire</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>d√©gressif</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="calculer">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>79</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(31, 29, 29);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>calculer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border:none ;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="timeIcon">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>169</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(22, 22, 22);
background:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/all/time.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="statIcon">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>169</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/all/bar-chart.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="materialIcon">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>174</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/all/processing.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="calendarIcon">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>169</y>
      <width>66</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleIcon">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/all/stat_.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="navigation">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent ;
border:none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="nom_materiel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background:white;
border:1px solid #dcdcdc;
border-radius:5px
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="duree_vie_materiel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>79</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background:white;
border:1px solid #dcdcdc;
border-radius:5px
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ajout">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>82</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(31, 29, 29);
color: rgb(255, 255, 255);
border-radius:20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ContactNew"/>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>974</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent ;
border:none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons/img.qrc"/>
 </resources>
 <connections/>
</ui>
